skaha/src/main/java/org/opencadc/skaha/utils/PosixHelper.java

```diff
@@ -17,15 +43,95 @@ public static String uidMapping(final PosixPrincipal posixPrincipal) {
                              posixPrincipal.getUidNumber());
     }
 
-    /**
-     * The name of the secret that contains the complete mapping of UIDs and GIDs.  This is mounted into the launched
-     * containers, and the init-users-groups.sh script uses it to populate the system files (/etc/passwd and
-     * /etc/group).
-     *
-     * @param sessionId The session identifier to mark the secret.
-     * @return String secret name, never null.
-     */
-    public static String getPosixMapperSecretName(String sessionId) {
-        return "posix-mapping-" + sessionId;
+    static Date currentDateUTC() {
+        return Calendar.getInstance(DateUtil.UTC).getTime();
+    }
+
+    static boolean posixMappingSecretRequiresUpdate() throws Exception {
```

Comment:
I guess these two methods and the executeInShell method aren't needed with this new approach

Reply:
The entire class can go, actually.